great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -285,6 +288,8 @@ def save_expectation_suite(
                 f"If you would like to overwrite this expectation_suite, set overwrite_existing=True."
             )
         self._evaluation_parameter_dependencies_compiled = False
+        if include_rendered_content:
+            expectation_suite.render()
```

Comment:
Is there value in making the method return a new suite with the updates? Or return itself? I'm okay with in-place too but just thinking of what is easier to reason about.

Reply:
This just populates a `rendered_content` attribute that starts as `None`. I don't see why we should create a copy and it already updates the pointer. Also, I like how pythonic these two lines are.